package.json

```diff
@@ -30,7 +30,6 @@
   },
   "lint-staged": {
     "src/**/*.+(js|jsx|ts|tsx)": [
-      "yarn lint",
       "yarn format",
```

Comment:
Not needed to do any changes except any new devDependencies are been used in the `package.json` file